### PR TITLE
Update projects' section

### PR DESCRIPTION
**Pull request by beatrizserrano**
I've tried my best, but I'm sure I'm missing parts of the picture. It would be nice to have your inputs @bgruening @erxleben @bebatut. Thanks!

### CODE REVIEW

**Comment by bgruening** [comment]
@bebatut should we add here OLS and StreetScience?

@beatrizserrano should we add Software Projects, like HiCExplorer and deepTools?

**Comment by bgruening** [comment]
I added some additional stuff. It would be nice if we could use some bootstrap cars (https://getbootstrap.com/docs/4.0/components/card/) or something similar to make this side fancier. But this is a good first step.

**Comment by bgruening** [comment]
Something like that?

1) https://getbootstrap.com/docs/5.0/examples/features/
2) https://getbootstrap.com/docs/5.0/examples/album/
3) https://getbootstrap.com/docs/5.0/examples/masonry/

File: freiburg/projects.md [hunk 1]
```diff
@@ -2,72 +2,167 @@
...
+<a name="hicexploer"></a>
+### HiCExplorer
+
+[HiCExplorer](https://hicexplorer.readthedocs.io/en/documentation/) is a set of programs to process, normalize, analyze and visualize Hi-C data. It addresses the common tasks of Hi-C analysis from processing to visualization. The Freiburg Galaxy team is involved in the maintenance and new development of HiCExplorer. You can access HiCExplorer via [https://hicexplorer.usegalaxy.eu](https://hicexplorer.usegalaxy.eu).
+
+<a name="deeptools"></a>
+### deepTools
+
+[deepTools](https://deeptools.readthedocs.io) is a suite of python tools particularly developed for the efficient analysis of high-throughput sequencing data, such as ChIP-seq, RNA-seq or MNase-seq. deepTools is the Swiss Army Knife of NGS processing and is cited
+more than 3000 times. It is of course part of the European Galaxy server and was used more than 230.000 times.
```
Comment (beatrizserrano) [inline freiburg/projects.md:145]:
```suggestion
more than 3,000 times. It is of course part of the European Galaxy server and was used more than 230,000 times.
```

**Comment by beatrizserrano** [comment]
>     2. https://getbootstrap.com/docs/5.0/examples/album/

this one maybe? The logos could fit there better I think

**Comment by bebatut** [comment]
I checked quickly the cards / album. We need to update bootstrap to use them as we are using a quite old bootstrap version (3)

**Comment by OlegZharkov** [comment]
> I checked quickly the cards / album. We need to update bootstrap to use them as we are using a quite old bootstrap version (3)

otherwise we can try this:
https://www.w3schools.com/howto/howto_css_cards.asp

**Comment by bgruening** [comment]
Sounds good Oleg. I would spent to much time on it since the migration to the new framework will probably happen this year as well.

**Comment by bgruening** [comment]
Will merge until we have something better, the text is an improvement.

Thanks everyone.

**Pull request merged**